R/viz.R

```diff
@@ -4828,29 +4861,32 @@ fancy_qc_plot = function(these_samples,
     this_meta = metadata
   }
 
+  if(!missing(these_samples_metadata)){
+    these_samples = dplyr::select(these_samples_metadata, sample_id) %>%
+      as.data.frame(strings.as.factors = FALSE) %>%
+      pull(sample_id)
+  }
+
   #filter metadata on selected cohort/pathology
-  if(missing(these_samples)){
+  if(missing(these_samples) && missing(these_samples_metadata)){
     if(!missing(filter_cohort) && missing(filter_pathology)){
```

Comment:
Should these be named "keep_cohort" or something that's more directionally descriptive? I don't know from the name if it means the cohort I want to keep or drop